### PR TITLE
Update tests and protobuf version

### PR DESCRIPTION
**Pull request by michaelbausor**
- Force protobuf version off broken 3.2.0 release
- Update tests with improved diff
- Update baseline files for protoc 3.2

### CODE REVIEW

File: test/test_plugin_baseline.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import pytest
 import subprocess
 import shutil
+import difflib
```
Comment (garrettjonesgoogle) [inline test/test_plugin_baseline.py]: nice :-)

File: test/test_plugin_baseline.py [hunk 2]
```diff
@@ -24,23 +25,35 @@
...
+               actual_output_file)
+
+
+def diff(expected_output, actual_output, fromfile, tofile):
+    difflines = list(difflib.context_diff(expected_output,
+                                          actual_output,
+                                          fromfile=fromfile,
+                                          tofile=tofile))
+    if len(difflines) > 50:
+        return "*** diff omitted: too long"
```
Comment (garrettjonesgoogle) [inline test/test_plugin_baseline.py]: Maybe we could return the "head" of the diff?

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 pystache>=0.5.4
-protobuf>=3.0.0
+protobuf==3.2.0rc2
```
Comment (pongad) [inline requirements.txt]: protobuf already release 3.2.0. Does python not have this package yet?

Comment (michaelbausor) [inline requirements.txt]: It is broken: https://github.com/google/protobuf/issues/2895

Comment (pongad) [inline requirements.txt]: Sigh

**Comment by pongad** [comment]
LGTM, but what's going on with Travis?

**Comment by michaelbausor** [comment]
Travis is not functional for this repo unfortunately. I will aim to resolve this as part of oneof work.

**Pull request merged**